README.md

```diff
@@ -48,13 +48,13 @@ Clone the repository
 
 - To set up a new project from the template, run the command:
 
-  - `$ make run PACKAGE_NAME={com.your.package} PROJECT_NAME={your_project_name} APP_NAME="{Your App Name}"`
+  - `$ make run PACKAGE_NAME={com.your.package} PROJECT_NAME={your_project_name} APP_NAME="{Your App Name}" APP_VERSION={your_app_version} BUILD_NUMBER={your_build_number}`
```

Comment:
> Should the build version/number stay as x.x.x instead of x_x_x?

The `app version` should be `x.x` or `x.x.x`. Let me update it in the table for clearer 👌 

> I think now that the command is long, we should NOT include the optional params. We want the devs to start as fast as possible with the default values, so how about listing the optional/mandatory fields in a table below instead of putting everything on this "Quick Start" example?

Agree, I will create a table and add all fields to that instead of listing them here 🙏